pandas/conftest.py

```diff
@@ -591,6 +593,91 @@ def narrow_series(request):
     return _narrow_series[request.param].copy()
 
 
+_dtype_nuisance_arr_map = {
```

Comment:
can you add a comment here about where this is used

Reply:
Done